README.md

```diff
@@ -1,5 +1,42 @@
+[![Haskell CI](https://img.shields.io/github/actions/workflow/status/input-output-hk/anti-diffs/haskell.yml?label=Build&style=for-the-badge)](https://github.com/input-output-hk/anti-diffs/actions/workflows/haskell.yml)
 [![handbook](https://img.shields.io/badge/policy-Cardano%20Engineering%20Handbook-informational?style=for-the-badge)](https://input-output-hk.github.io/cardano-engineering-handbook)
 
 # anti-diffs
 
-Packages for sequences of `Data.Map` differences.
\ No newline at end of file
+Packages for performant sequences of `Data.Map` differences.
+
+Important documents:
+* [CONTRIBUTING.md](./CONTRIBUTING.md)
+* [SECURITY.md](./SECURITY.md)
+* [CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md)
+* [ouroboros-network/README.md](
+https://github.com/input-output-hk/ouroboros-network/blob/master/README.md). We
+follow processes and guidelines as established by the Consensus team.
```

Comment:
I don't see the point in listing these as "Important documents". I would not really mention them, most of the people know what they are already, except for the last one. And in the case of the last one, it is pointing to the README of ouroboros-network in which we don't really say much about the consensus team

Reply:
I've removed the list. I did keep the last one on the list but I changed the `ouroboros-network/README.md` hyperlink to [`ouroboros-network/ouroboros-consensus/README.md`](https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/README.md), since we use the same styleguide/versioning policy/changelog policies as consensus